src/validator/azd_validator.py

```diff
@@ -2,18 +2,23 @@
 import subprocess
 import logging
 from validator.validator_base import ValidatorBase
-from constants import *
+from constants import ItemResultFormat, line_delimiter
 from utils import indent, retry
 
 # defines string array of retryable error messages
-retryable_error_messages = [
-    "Cannot connect to the Docker daemon",
-    "spawn ETXTBSY"
-]
+retryable_error_messages = ["Cannot connect to the Docker daemon", "spawn ETXTBSY"]
+
 
 class AzdValidator(ValidatorBase):
-    def __init__(self, validatorCatalog, folderPath, check_azd_up=True, check_azd_down=True, errorAsWarning=False):
-        super().__init__(f"AzdValidator", validatorCatalog, errorAsWarning)
+    def __init__(
+        self,
+        validatorCatalog,
+        folderPath,
+        check_azd_up=True,
```

Comment:
Is it possible set an enum about azd action here ?
Currently, including up/down
Validator only run one action